molecule/default/tests/test_default.py

```diff
@@ -1,25 +1,124 @@
 import os
 import re
-
 import pytest
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
     os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')
 
 
-@pytest.mark.skip(reason='TODO: will update the test to run '
-                         'cinder command on controller')
+@pytest.mark.jira('asc-222')
+@pytest.mark.skip(reason='this test can not run on storage host')
+# TODO: will update the test to run cinder command on controller node
+# TODO: since the test cannot run on storage hosts.
 def test_cinder_service(host):
+    """Test to verify that cinder service is running on the cinder nodes
+
+    Args:
+        host(Testinfra host fixture): A hostname in dynamic_inventory.json/molecule.yml
+    """
+
     cmd = "sudo bash -c \"source /root/openrc; cinder service-list\""
     output = host.run(cmd)
     assert ("cinder-volume" in output.stdout)
 
 
+@pytest.mark.jira('asc-222')
 def test_cinder_lvm_volume(host):
+    """Test 2a: Check the Cinder Nodes: volume group
+
+    Args:
+        host(Testinfra host fixture): A hostname in dynamic_inventory.json/molecule.yml
+    """
+
     output = host.run('vgs cinder-volumes')
     assert re.search("cinder-volumes\s+[0-9]*\s+[0-9]*\s+", output.stdout)
 
 
+@pytest.mark.jira('asc-222')
 def test_cinder_volume_group(host):
+    """Test 2a: Check the Cinder config file
+
+    Args:
+        host(Testinfra host fixture): A hostname in dynamic_inventory.json/molecule.yml
+    """
+
     assert host.file('/etc/cinder/cinder.conf').contains("volume_group")
+
+
+@pytest.mark.jira('asc-222')
+@pytest.mark.skip(reason='This test is only for bare metal')
+# TODO: 1. This test is only for bare metal
+# TODO: 2. This test must be executed on compute_hosts (not storage_hosts)
+def test_list_lxc_volume_group(host):
+    """Test 2b:
+    Check Compute hosts LVM setup and verify we're not leaving an excess of free extents.
+    Notes: makes sure if you are using LVM for your partitioning, that all
+    available disk space, past the amount needed for the OS, has been allocated
+    to /var/lib/nova
+
+    Args:
+        host(Testinfra host fixture): A hostname in dynamic_inventory.json/molecule.yml
```

Comment:
A small nit pick:

This should be `host(testinfra.host.Host): A hostname in dynamic_inventory.json/molecule.yml`